nilearn/plotting/html_stat_map.py

```diff
@@ -0,0 +1,97 @@
+import os
+import tempfile
+import base64
+
+import numpy as np
+from nilearn import image, datasets
+
+from .._utils.extmath import fast_abs_percentile
+from .._utils.param_validation import check_threshold
+from .js_plotting_utils import get_html_template, HTMLDocument
+
+
+def _encode_nii(img):
```

Comment:
Does papaya read all dtypes? I have in mind that it doesn't, including pretty common dtypes (like one the float32 / float64)